core/storage/suggestion/gae_models_test.py

```diff
@@ -93,6 +93,66 @@ def setUp(self) -> None:
             'reviewer_2', self.change_cmd, self.score_category,
             'exploration.exp1.thread_5', None)
 
+    def test_get_model_association_to_user(self) -> None:
+        gen_model = suggestion_models.GeneralSuggestionModel
+        self.assertEqual(
+            gen_model.get_model_association_to_user(),
+            base_models.MODEL_ASSOCIATION_TO_USER.MULTIPLE_INSTANCES_PER_USER
+        )
+
+    def test_get_export_policy(self) -> None:
+        dictionary = {
+            'suggestion_type': base_models.EXPORT_POLICY.EXPORTED,
+            'target_type': base_models.EXPORT_POLICY.EXPORTED,
+            'target_id': base_models.EXPORT_POLICY.EXPORTED,
+            'target_version_at_submission':
+                base_models.EXPORT_POLICY.EXPORTED,
+            'deleted': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+            'last_updated': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+            'status': base_models.EXPORT_POLICY.EXPORTED,
+            'author_id': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+            'final_reviewer_id': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+            'change_cmd': base_models.EXPORT_POLICY.EXPORTED,
+            'score_category': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+            'created_on': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+            'language_code': base_models.EXPORT_POLICY.EXPORTED,
+            'edited_by_reviewer': base_models.EXPORT_POLICY.EXPORTED
+        }
+        self.assertEqual(
+            suggestion_models.GeneralSuggestionModel.get_export_policy(),
+            dictionary
+        )
+
+    def test_get_in_review_translation_suggestions(self) -> None:
+        gen = suggestion_models.GeneralSuggestionModel
```

Comment:
Do not use names like `gen`. No one knows what that means.

Instead, just call it GeneralSuggestionModel or suggestion_model_class or similar. Always use descriptive names that are self-evidently clear.

Ditto elsewhere.

Reply:
Resolved this. Although in one line (189), due to the long method name, I disabled pylint for that line only.